### PR TITLE
Story/ccls 1716 equal opportunities monitoring screen

### PR DESCRIPTION
**Pull request by PhilDigitalJustice**

### CODE REVIEW

File: src/main/java/uk/gov/laa/ccms/caab/controller/application/client/ClientEqualOpportunitiesMonitoringDetailsController.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.SessionAttributes;
+import reactor.core.publisher.Mono;
+import uk.gov.laa.ccms.caab.bean.ClientDetails;
+import uk.gov.laa.ccms.caab.bean.validators.client.ClientEqualOpportunitiesMonitoringDetailsValidator;
+import uk.gov.laa.ccms.caab.service.CommonLookupService;
+import uk.gov.laa.ccms.data.model.CommonLookupDetail;
+
+/**
+ * Controller for handling contact client details selection during the new application process.
```
Comment (porritta) [inline src/main/java/uk/gov/laa/ccms/caab/controller/application/client/ClientEqualOpportunitiesMonitoringDetailsController.java:21]: Update javadocs in this class, guess it's copied from elsewhere

File: src/main/resources/templates/application/client/equal-opportunities-monitoring-client-details.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+      </p>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <div class="govuk-form-group" th:classappend="${#fields.hasErrors('ethnicOrigin')} ? 'govuk-form-group--error' : ''">
+            <h2 class="govuk-heading-m">
+              Ethnic monitoring
+            </h2>
+            <fieldset class="govuk-fieldset">
+              <label class="govuk-label" for="ethnicOrigin">
+                Which ethnicity would your client describe themselves being?
```
Comment (porritta) [inline src/main/resources/templates/application/client/equal-opportunities-monitoring-client-details.html:29]: "as being"?

**Review by porritta — COMMENTED** [review]
Just a couple of minor comments

File: src/main/resources/templates/application/client/equal-opportunities-monitoring-client-details.html (hunk 2 above)
Comment (PhilDigitalJustice) [reply src/main/resources/templates/application/client/equal-opportunities-monitoring-client-details.html]: Amended

File: src/main/java/uk/gov/laa/ccms/caab/controller/application/client/ClientEqualOpportunitiesMonitoringDetailsController.java (hunk 1 above)
Comment (PhilDigitalJustice) [reply src/main/java/uk/gov/laa/ccms/caab/controller/application/client/ClientEqualOpportunitiesMonitoringDetailsController.java]: amended

**Review by porritta — APPROVED** [review]
LGTM

**Pull request merged**